TestControllers/HyperVController.psm1

```diff
@@ -135,5 +135,6 @@ Class HyperVController : TestController
 		([TestController]$this).SetGlobalVariables()
 
 		Set-Variable -Name VMGeneration -Value $this.TestProvider.VMGeneration -Scope Global
+		Set-Variable -Name VMIntegrationKeyValuePairExchange -Value "Key-Value Pair Exchange" -Scope Global
```

Comment:
I see that this variable is only used in one test script.
If yes, then what is the need of global variable?

Reply:
The variable is only used in one test script, but the "Key-Value Pair Exchange" is used in several scripts. All the "Key-Value Pair Exchange" will be replaced with the global variable soon. 
 
Juhlee-microsoft's advice: For the integration service name such as "Key-Value Pair Exchange", "Guest Service Interface", etc., there is a chance the service names change. Replace them with the global constant in the file. So we will update one place with new service name in future.